src/appkit.cs

```diff
@@ -23562,14 +23566,14 @@ interface NSFilePromiseProvider : NSPasteboardWriting
 	interface NSFilePromiseProviderDelegate
```

Comment:
if the two changes here are not breaking changes then it means it's a new type in 10.12 and there should be an availability attribute on the type

it also should be marked as a `[Protocol]` and `[Model]` and anything that use it should do so from the interface that represent the protocol (and not the concrete type that the model provides)